# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/service/VRService.h
# Commit: 1d9384906287
# Full Hash: 1d93849062870a8d492eeef2150ecea174cb6223
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r f5ebdcca640d -r 1d9384906287 gfx/vr/service/VRService.h
--- a/gfx/vr/service/VRService.h	Sun May 26 14:29:56 2019 +0000
+++ b/gfx/vr/service/VRService.h	Sun May 26 14:30:14 2019 +0000
@@ -66,10 +66,7 @@
 #if defined(XP_WIN)
   HANDLE mMutex;
 #endif
-  // We store the value of gfxPrefs::VRProcessEnabled() in mVRProcessEnabled.
-  // This allows us to read the value in the VRService destructor, after
-  // gfxPrefs has been shut down.  We should investigate why gfxPrefs
-  // is shutting down earlier - See bug xxx
+  // We store the value of StaticPrefs::VRProcessEnabled() in mVRProcessEnabled.
   bool mVRProcessEnabled;
 
   bool IsInServiceThread();